[E]	manager_11@10.16.0.247	2018-10-18 17:00:04.385618 +0800	1539853204	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:04.386341 +0800	1539853204	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:04.386526 +0800	1539853204	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.30904.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:00:09.387541 +0800	1539853209	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:09.388118 +0800	1539853209	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:09.416114 +0800	1539853209	null:null	0	CRASH REPORT Process <0.31003.5> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:00:09.416509 +0800	1539853209	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:00:09.416649 +0800	1539853209	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.30952.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:00:09.416842 +0800	1539853209	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:00:14.417618 +0800	1539853214	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:14.417909 +0800	1539853214	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:14.418062 +0800	1539853214	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31005.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:00:19.419594 +0800	1539853219	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:19.419995 +0800	1539853219	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:19.420230 +0800	1539853219	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31051.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:00:24.421593 +0800	1539853224	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:24.422093 +0800	1539853224	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:24.422309 +0800	1539853224	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31102.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:00:29.423522 +0800	1539853229	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:29.424057 +0800	1539853229	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:29.454954 +0800	1539853229	null:null	0	CRASH REPORT Process <0.31174.5> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:00:29.455329 +0800	1539853229	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:00:29.455509 +0800	1539853229	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31148.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:00:29.455623 +0800	1539853229	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:00:34.456588 +0800	1539853234	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:34.456923 +0800	1539853234	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:34.457076 +0800	1539853234	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31176.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:00:39.457605 +0800	1539853239	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:39.457912 +0800	1539853239	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:39.458047 +0800	1539853239	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31216.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:00:44.459677 +0800	1539853244	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:44.459995 +0800	1539853244	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:44.460121 +0800	1539853244	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31305.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:00:49.461536 +0800	1539853249	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:49.461817 +0800	1539853249	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:49.494547 +0800	1539853249	null:null	0	CRASH REPORT Process <0.31400.5> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:00:49.494899 +0800	1539853249	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:00:49.495039 +0800	1539853249	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31348.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:00:49.495160 +0800	1539853249	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:00:54.496608 +0800	1539853254	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:54.497101 +0800	1539853254	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:54.497346 +0800	1539853254	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31402.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:00:59.498624 +0800	1539853259	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:00:59.499079 +0800	1539853259	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:00:59.499303 +0800	1539853259	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31449.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:01:04.500564 +0800	1539853264	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:04.501067 +0800	1539853264	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:04.501273 +0800	1539853264	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31499.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:01:09.502539 +0800	1539853269	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:09.503066 +0800	1539853269	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:09.533030 +0800	1539853269	null:null	0	CRASH REPORT Process <0.31559.5> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:01:09.533269 +0800	1539853269	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:01:09.533402 +0800	1539853269	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31520.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:01:09.533504 +0800	1539853269	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:01:14.534634 +0800	1539853274	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:14.534981 +0800	1539853274	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:14.535127 +0800	1539853274	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31561.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:01:19.536619 +0800	1539853279	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:19.536997 +0800	1539853279	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:19.537133 +0800	1539853279	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31614.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:01:24.538635 +0800	1539853284	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:24.538971 +0800	1539853284	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:24.539115 +0800	1539853284	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31655.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:01:29.540585 +0800	1539853289	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:29.540935 +0800	1539853289	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:29.571139 +0800	1539853289	null:null	0	CRASH REPORT Process <0.31752.5> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:01:29.571347 +0800	1539853289	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:01:29.571453 +0800	1539853289	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31709.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:01:29.571557 +0800	1539853289	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:01:34.572628 +0800	1539853294	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:34.572914 +0800	1539853294	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:34.573062 +0800	1539853294	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31754.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:01:39.573567 +0800	1539853299	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:39.573981 +0800	1539853299	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:39.574212 +0800	1539853299	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31809.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:01:44.575567 +0800	1539853304	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:44.576069 +0800	1539853304	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:44.576274 +0800	1539853304	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31871.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:01:49.577535 +0800	1539853309	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:49.578112 +0800	1539853309	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:49.611187 +0800	1539853309	null:null	0	CRASH REPORT Process <0.31936.5> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:01:49.611592 +0800	1539853309	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:01:49.611784 +0800	1539853309	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31891.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:01:49.611896 +0800	1539853309	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:01:54.612607 +0800	1539853314	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:54.613053 +0800	1539853314	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:54.613279 +0800	1539853314	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31938.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:01:59.614618 +0800	1539853319	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:01:59.615155 +0800	1539853319	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:01:59.615383 +0800	1539853319	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32082.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:02:04.616607 +0800	1539853324	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:04.617106 +0800	1539853324	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:04.617333 +0800	1539853324	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32174.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:02:09.618529 +0800	1539853329	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:09.619224 +0800	1539853329	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:09.649465 +0800	1539853329	null:null	0	CRASH REPORT Process <0.32294.5> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:02:09.649820 +0800	1539853329	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:02:09.649944 +0800	1539853329	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32226.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:02:09.650050 +0800	1539853329	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:02:14.651572 +0800	1539853334	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:14.652018 +0800	1539853334	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:14.652248 +0800	1539853334	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32296.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:02:19.653563 +0800	1539853339	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:19.654038 +0800	1539853339	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:19.654248 +0800	1539853339	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32341.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:02:24.655566 +0800	1539853344	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:24.656062 +0800	1539853344	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:24.656268 +0800	1539853344	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32379.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:02:29.657533 +0800	1539853349	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:29.658083 +0800	1539853349	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:29.688400 +0800	1539853349	null:null	0	CRASH REPORT Process <0.32505.5> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:02:29.688781 +0800	1539853349	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:02:29.688939 +0800	1539853349	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32435.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:02:29.689053 +0800	1539853349	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:02:34.690641 +0800	1539853354	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:34.691049 +0800	1539853354	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:34.691200 +0800	1539853354	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32507.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:02:39.692550 +0800	1539853359	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:39.692857 +0800	1539853359	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:39.692992 +0800	1539853359	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32561.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:02:44.694576 +0800	1539853364	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:44.695019 +0800	1539853364	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:44.695235 +0800	1539853364	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32647.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:02:49.696556 +0800	1539853369	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:49.697077 +0800	1539853369	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:49.726602 +0800	1539853369	null:null	0	CRASH REPORT Process <0.32728.5> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:02:49.726874 +0800	1539853369	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:02:49.726998 +0800	1539853369	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32701.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:02:49.727106 +0800	1539853369	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:02:54.727682 +0800	1539853374	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:54.728059 +0800	1539853374	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:54.728214 +0800	1539853374	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32730.5> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:02:59.729614 +0800	1539853379	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:02:59.729963 +0800	1539853379	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:02:59.730104 +0800	1539853379	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.22.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:03:04.731673 +0800	1539853384	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:04.732031 +0800	1539853384	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:04.732178 +0800	1539853384	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.67.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:03:09.733535 +0800	1539853389	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:09.733839 +0800	1539853389	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:09.764669 +0800	1539853389	null:null	0	CRASH REPORT Process <0.181.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:03:09.764881 +0800	1539853389	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:03:09.765007 +0800	1539853389	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.133.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:03:09.765129 +0800	1539853389	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:03:14.766666 +0800	1539853394	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:14.767101 +0800	1539853394	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:14.767348 +0800	1539853394	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.183.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:03:19.768574 +0800	1539853399	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:19.769067 +0800	1539853399	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:19.769274 +0800	1539853399	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.255.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:03:24.770601 +0800	1539853404	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:24.771102 +0800	1539853404	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:24.771319 +0800	1539853404	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.297.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:03:29.772536 +0800	1539853409	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:29.773102 +0800	1539853409	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:29.803160 +0800	1539853409	null:null	0	CRASH REPORT Process <0.389.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:03:29.803518 +0800	1539853409	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:03:29.803679 +0800	1539853409	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.364.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:03:29.803801 +0800	1539853409	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:03:34.804701 +0800	1539853414	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:34.805042 +0800	1539853414	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:34.805188 +0800	1539853414	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.391.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:03:39.806652 +0800	1539853419	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:39.806992 +0800	1539853419	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:39.807142 +0800	1539853419	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.453.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:03:44.808676 +0800	1539853424	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:44.809118 +0800	1539853424	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:44.809341 +0800	1539853424	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.528.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:03:49.810534 +0800	1539853429	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:49.810815 +0800	1539853429	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:49.842559 +0800	1539853429	null:null	0	CRASH REPORT Process <0.641.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:03:49.842854 +0800	1539853429	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:03:49.842973 +0800	1539853429	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.590.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:03:49.843093 +0800	1539853429	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:03:54.844601 +0800	1539853434	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:54.844951 +0800	1539853434	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:54.845115 +0800	1539853434	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.643.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:03:59.846596 +0800	1539853439	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:03:59.846982 +0800	1539853439	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:03:59.847129 +0800	1539853439	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.708.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:04:04.848682 +0800	1539853444	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:04.849110 +0800	1539853444	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:04.849353 +0800	1539853444	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.753.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:04:09.850548 +0800	1539853449	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:09.851092 +0800	1539853449	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:09.880393 +0800	1539853449	null:null	0	CRASH REPORT Process <0.843.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:04:09.880734 +0800	1539853449	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:04:09.880859 +0800	1539853449	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.791.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:04:09.880968 +0800	1539853449	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:04:14.881692 +0800	1539853454	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:14.882105 +0800	1539853454	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:14.882243 +0800	1539853454	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.845.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:04:19.883665 +0800	1539853459	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:19.884082 +0800	1539853459	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:19.884326 +0800	1539853459	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.914.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:04:24.885652 +0800	1539853464	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:24.886145 +0800	1539853464	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:24.886366 +0800	1539853464	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.965.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:04:29.887536 +0800	1539853469	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:29.888042 +0800	1539853469	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:29.919071 +0800	1539853469	null:null	0	CRASH REPORT Process <0.1085.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:04:29.919410 +0800	1539853469	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:04:29.919550 +0800	1539853469	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1028.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:04:29.919677 +0800	1539853469	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:04:34.920645 +0800	1539853474	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:34.921114 +0800	1539853474	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:34.921263 +0800	1539853474	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1087.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:04:39.922591 +0800	1539853479	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:39.922945 +0800	1539853479	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:39.923091 +0800	1539853479	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1149.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:04:44.924671 +0800	1539853484	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:44.925132 +0800	1539853484	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:44.925372 +0800	1539853484	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1199.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:04:49.926522 +0800	1539853489	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:49.927055 +0800	1539853489	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:49.955532 +0800	1539853489	null:null	0	CRASH REPORT Process <0.1309.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:04:49.955903 +0800	1539853489	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:04:49.956037 +0800	1539853489	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1255.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:04:49.956143 +0800	1539853489	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:04:54.957669 +0800	1539853494	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:54.958112 +0800	1539853494	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:54.958356 +0800	1539853494	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1311.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:04:59.959610 +0800	1539853499	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:04:59.960144 +0800	1539853499	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:04:59.960372 +0800	1539853499	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1374.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:05:04.961679 +0800	1539853504	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:05:04.962189 +0800	1539853504	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:05:04.962417 +0800	1539853504	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1425.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:05:09.963525 +0800	1539853509	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:05:09.964067 +0800	1539853509	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:05:09.992836 +0800	1539853509	null:null	0	CRASH REPORT Process <0.1537.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:05:09.993195 +0800	1539853509	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:05:09.993368 +0800	1539853509	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1489.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:05:09.993481 +0800	1539853509	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:05:14.994686 +0800	1539853514	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:05:14.995061 +0800	1539853514	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:05:14.995205 +0800	1539853514	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1539.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:05:19.996669 +0800	1539853519	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:05:19.997002 +0800	1539853519	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:05:19.997157 +0800	1539853519	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1562.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:05:24.998645 +0800	1539853524	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:05:24.999080 +0800	1539853524	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:05:24.999217 +0800	1539853524	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1630.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:05:29.999527 +0800	1539853529	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:05:29.999831 +0800	1539853529	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:05:30.32550 +0800	1539853530	null:null	0	CRASH REPORT Process <0.1753.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:05:30.32880 +0800	1539853530	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:05:30.32988 +0800	1539853530	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1681.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:05:30.33102 +0800	1539853530	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:05:35.34679 +0800	1539853535	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:05:35.35088 +0800	1539853535	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:05:35.35229 +0800	1539853535	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1756.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:05:40.36671 +0800	1539853540	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:05:40.37005 +0800	1539853540	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:05:40.37147 +0800	1539853540	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1804.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:05:45.38641 +0800	1539853545	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:05:45.39154 +0800	1539853545	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:05:45.39380 +0800	1539853545	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1909.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:05:50.40520 +0800	1539853550	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:05:50.41079 +0800	1539853550	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:05:50.69695 +0800	1539853550	null:null	0	CRASH REPORT Process <0.2010.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:05:50.70048 +0800	1539853550	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:05:50.70216 +0800	1539853550	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.1958.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:05:50.70348 +0800	1539853550	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:05:55.71650 +0800	1539853555	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:05:55.72060 +0800	1539853555	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:05:55.72213 +0800	1539853555	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2012.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:06:00.73653 +0800	1539853560	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:00.74131 +0800	1539853560	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:00.74345 +0800	1539853560	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2048.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:06:05.75665 +0800	1539853565	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:05.75997 +0800	1539853565	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:05.76142 +0800	1539853565	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2114.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:06:10.77551 +0800	1539853570	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:10.77866 +0800	1539853570	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:10.107914 +0800	1539853570	null:null	0	CRASH REPORT Process <0.2227.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:06:10.108228 +0800	1539853570	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:06:10.108351 +0800	1539853570	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2164.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:06:10.108467 +0800	1539853570	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:06:15.109607 +0800	1539853575	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:15.110035 +0800	1539853575	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:15.110195 +0800	1539853575	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2229.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:06:20.111635 +0800	1539853580	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:20.111919 +0800	1539853580	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:20.112060 +0800	1539853580	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2281.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:06:25.113659 +0800	1539853585	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:25.114144 +0800	1539853585	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:25.114371 +0800	1539853585	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2338.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:06:30.115533 +0800	1539853590	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:30.116171 +0800	1539853590	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:30.144638 +0800	1539853590	null:null	0	CRASH REPORT Process <0.2431.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:06:30.145019 +0800	1539853590	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:06:30.145177 +0800	1539853590	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2365.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:06:30.145322 +0800	1539853590	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:06:35.146644 +0800	1539853595	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:35.147023 +0800	1539853595	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:35.147176 +0800	1539853595	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2433.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:06:40.148637 +0800	1539853600	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:40.148925 +0800	1539853600	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:40.149073 +0800	1539853600	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2486.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:06:45.150632 +0800	1539853605	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:45.151092 +0800	1539853605	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:45.151333 +0800	1539853605	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2570.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:06:50.152537 +0800	1539853610	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:50.152886 +0800	1539853610	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:50.182330 +0800	1539853610	null:null	0	CRASH REPORT Process <0.2689.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:06:50.182693 +0800	1539853610	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:06:50.182821 +0800	1539853610	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2624.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:06:50.182936 +0800	1539853610	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:06:55.183632 +0800	1539853615	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:06:55.184086 +0800	1539853615	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:06:55.184326 +0800	1539853615	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2691.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:07:00.185600 +0800	1539853620	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:00.186088 +0800	1539853620	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:00.186304 +0800	1539853620	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2744.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:07:05.187588 +0800	1539853625	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:05.188092 +0800	1539853625	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:05.188318 +0800	1539853625	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2784.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:07:10.189558 +0800	1539853630	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:10.190111 +0800	1539853630	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:10.221168 +0800	1539853630	null:null	0	CRASH REPORT Process <0.2890.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:07:10.221534 +0800	1539853630	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:07:10.221656 +0800	1539853630	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2827.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:07:10.221768 +0800	1539853630	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:07:15.222647 +0800	1539853635	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:15.223012 +0800	1539853635	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:15.223161 +0800	1539853635	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2893.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:07:20.224660 +0800	1539853640	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:20.224942 +0800	1539853640	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:20.225089 +0800	1539853640	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.2946.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:07:25.226579 +0800	1539853645	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:25.227030 +0800	1539853645	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:25.227267 +0800	1539853645	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3008.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:07:30.228546 +0800	1539853650	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:30.229097 +0800	1539853650	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:30.259644 +0800	1539853650	null:null	0	CRASH REPORT Process <0.3114.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:07:30.259832 +0800	1539853650	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:07:30.259974 +0800	1539853650	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3063.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:07:30.260090 +0800	1539853650	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:07:35.261577 +0800	1539853655	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:35.262013 +0800	1539853655	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:35.262241 +0800	1539853655	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3122.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:07:40.263649 +0800	1539853660	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:40.264171 +0800	1539853660	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:40.264391 +0800	1539853660	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3154.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:07:45.265610 +0800	1539853665	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:45.266132 +0800	1539853665	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:45.266358 +0800	1539853665	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3221.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:07:50.267524 +0800	1539853670	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:50.268057 +0800	1539853670	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:50.296895 +0800	1539853670	null:null	0	CRASH REPORT Process <0.3340.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:07:50.297267 +0800	1539853670	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:07:50.297440 +0800	1539853670	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3291.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:07:50.297554 +0800	1539853670	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:07:55.298609 +0800	1539853675	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:07:55.299014 +0800	1539853675	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:07:55.299173 +0800	1539853675	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3342.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:08:00.300606 +0800	1539853680	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:00.300969 +0800	1539853680	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:00.301121 +0800	1539853680	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3411.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:08:05.302559 +0800	1539853685	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:05.302994 +0800	1539853685	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:05.303226 +0800	1539853685	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3458.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:08:10.303548 +0800	1539853690	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:10.304063 +0800	1539853690	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:10.334440 +0800	1539853690	null:null	0	CRASH REPORT Process <0.3546.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:08:10.334724 +0800	1539853690	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:08:10.334830 +0800	1539853690	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3523.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:08:10.334929 +0800	1539853690	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:08:15.335654 +0800	1539853695	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:15.335991 +0800	1539853695	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:15.336139 +0800	1539853695	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3548.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:08:20.337688 +0800	1539853700	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:20.338065 +0800	1539853700	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:20.338210 +0800	1539853700	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3621.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:08:25.339557 +0800	1539853705	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:25.339919 +0800	1539853705	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:25.340073 +0800	1539853705	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3664.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:08:30.341536 +0800	1539853710	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:30.341801 +0800	1539853710	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:30.372501 +0800	1539853710	null:null	0	CRASH REPORT Process <0.3788.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:08:30.372837 +0800	1539853710	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:08:30.373004 +0800	1539853710	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3735.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:08:30.373121 +0800	1539853710	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:08:35.374627 +0800	1539853715	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:35.374925 +0800	1539853715	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:35.375081 +0800	1539853715	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3790.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:08:40.376613 +0800	1539853720	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:40.376950 +0800	1539853720	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:40.377102 +0800	1539853720	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3851.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:08:45.378652 +0800	1539853725	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:45.379069 +0800	1539853725	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:45.379302 +0800	1539853725	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3922.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:08:50.380543 +0800	1539853730	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:50.381101 +0800	1539853730	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:50.409981 +0800	1539853730	null:null	0	CRASH REPORT Process <0.4007.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:08:50.410301 +0800	1539853730	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:08:50.410420 +0800	1539853730	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.3959.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:08:50.410524 +0800	1539853730	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:08:55.411676 +0800	1539853735	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:08:55.412106 +0800	1539853735	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:08:55.412350 +0800	1539853735	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4009.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:09:00.413634 +0800	1539853740	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:00.414134 +0800	1539853740	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:00.414355 +0800	1539853740	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4075.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:09:05.415634 +0800	1539853745	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:05.416140 +0800	1539853745	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:05.416363 +0800	1539853745	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4128.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:09:10.417532 +0800	1539853750	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:10.418057 +0800	1539853750	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:10.447959 +0800	1539853750	null:null	0	CRASH REPORT Process <0.4247.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:09:10.448340 +0800	1539853750	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:09:10.448500 +0800	1539853750	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4193.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:09:10.448616 +0800	1539853750	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:09:15.449591 +0800	1539853755	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:15.449927 +0800	1539853755	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:15.450085 +0800	1539853755	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4249.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:09:20.451639 +0800	1539853760	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:20.452036 +0800	1539853760	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:20.452259 +0800	1539853760	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4425.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:09:25.453616 +0800	1539853765	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:25.454943 +0800	1539853765	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:25.455164 +0800	1539853765	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4516.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:09:30.455594 +0800	1539853770	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:30.456143 +0800	1539853770	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:30.486906 +0800	1539853770	null:null	0	CRASH REPORT Process <0.4611.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:09:30.487111 +0800	1539853770	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:09:30.487236 +0800	1539853770	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4558.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:09:30.487381 +0800	1539853770	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:09:35.488615 +0800	1539853775	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:35.488923 +0800	1539853775	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:35.489085 +0800	1539853775	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4613.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:09:40.490632 +0800	1539853780	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:40.490999 +0800	1539853780	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:40.491140 +0800	1539853780	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4662.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:09:45.492619 +0800	1539853785	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:45.492942 +0800	1539853785	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:45.493108 +0800	1539853785	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4729.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:09:50.494514 +0800	1539853790	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:50.494827 +0800	1539853790	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:50.525107 +0800	1539853790	null:null	0	CRASH REPORT Process <0.4800.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:09:50.525321 +0800	1539853790	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:09:50.525429 +0800	1539853790	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4772.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:09:50.525531 +0800	1539853790	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:09:55.526625 +0800	1539853795	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:09:55.527000 +0800	1539853795	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:09:55.527150 +0800	1539853795	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4808.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:10:00.528689 +0800	1539853800	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:00.529026 +0800	1539853800	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:00.529157 +0800	1539853800	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4841.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:10:05.530611 +0800	1539853805	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:05.531517 +0800	1539853805	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:05.531743 +0800	1539853805	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4889.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:10:10.531547 +0800	1539853810	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:10.532072 +0800	1539853810	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:10.563499 +0800	1539853810	null:null	0	CRASH REPORT Process <0.4989.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:10:10.563832 +0800	1539853810	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:10:10.563958 +0800	1539853810	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4936.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:10:10.564080 +0800	1539853810	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:10:15.564613 +0800	1539853815	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:15.565098 +0800	1539853815	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:15.565277 +0800	1539853815	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.4991.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:10:20.566580 +0800	1539853820	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:20.567189 +0800	1539853820	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:20.567369 +0800	1539853820	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5038.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:10:25.568600 +0800	1539853825	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:25.569190 +0800	1539853825	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:25.569374 +0800	1539853825	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5087.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:10:30.570535 +0800	1539853830	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:30.571070 +0800	1539853830	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:30.601390 +0800	1539853830	null:null	0	CRASH REPORT Process <0.5153.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:10:30.601687 +0800	1539853830	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:10:30.601804 +0800	1539853830	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5119.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:10:30.601911 +0800	1539853830	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:10:35.602604 +0800	1539853835	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:35.602923 +0800	1539853835	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:35.603071 +0800	1539853835	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5156.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:10:40.604666 +0800	1539853840	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:40.604975 +0800	1539853840	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:40.605112 +0800	1539853840	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5203.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:10:45.606610 +0800	1539853845	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:45.606992 +0800	1539853845	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:45.607143 +0800	1539853845	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5293.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:10:50.608537 +0800	1539853850	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:50.608815 +0800	1539853850	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:50.640206 +0800	1539853850	null:null	0	CRASH REPORT Process <0.5392.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:10:50.640397 +0800	1539853850	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:10:50.640511 +0800	1539853850	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5342.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:10:50.640633 +0800	1539853850	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:10:55.641658 +0800	1539853855	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:10:55.642015 +0800	1539853855	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:10:55.642148 +0800	1539853855	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5394.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:11:00.643579 +0800	1539853860	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:11:00.643893 +0800	1539853860	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:11:00.644019 +0800	1539853860	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5441.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:11:05.645652 +0800	1539853865	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:11:05.646147 +0800	1539853865	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:11:05.646354 +0800	1539853865	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5479.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:11:10.647534 +0800	1539853870	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:11:10.648069 +0800	1539853870	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:11:10.678237 +0800	1539853870	null:null	0	CRASH REPORT Process <0.5550.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:11:10.678632 +0800	1539853870	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:11:10.678808 +0800	1539853870	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5505.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:11:10.678922 +0800	1539853870	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:11:15.679633 +0800	1539853875	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:11:15.679993 +0800	1539853875	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:11:15.680137 +0800	1539853875	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5552.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:11:20.681609 +0800	1539853880	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:11:20.681942 +0800	1539853880	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:11:20.682082 +0800	1539853880	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5607.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:11:25.683622 +0800	1539853885	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:11:25.684141 +0800	1539853885	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:11:25.684342 +0800	1539853885	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5651.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:11:30.685543 +0800	1539853890	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:11:30.686050 +0800	1539853890	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:11:30.716852 +0800	1539853890	null:null	0	CRASH REPORT Process <0.5767.6> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 17:11:30.717060 +0800	1539853890	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 17:11:30.717186 +0800	1539853890	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5701.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 17:11:30.717316 +0800	1539853890	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 17:11:35.718604 +0800	1539853895	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:11:35.718918 +0800	1539853895	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:11:35.719105 +0800	1539853895	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5770.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:11:40.720598 +0800	1539853900	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:11:40.721108 +0800	1539853900	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:11:40.721306 +0800	1539853900	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5816.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 17:11:45.722605 +0800	1539853905	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 17:11:45.722909 +0800	1539853905	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 17:11:45.723043 +0800	1539853905	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.5858.6> exit with reason no such process or port in call to gen_server:call(<0.1392.0>, raw_status) in context child_terminated
